src/main/java/domain/Number.java

```diff
@@ -0,0 +1,35 @@
+package domain;
+
+import java.util.Objects;
+
+public class Number {
+    private final int number;
+
+    public Number(int number) {
+        validate(number);
+        this.number = number;
+    }
+
+    private void validate(int number) {
+        if (!(1 <= number && number <= 45)) {
```

Comment:
```suggestion
        if (number < 1 || number > 45) {
```
조건문을 정의할때는 항상 일관적인 규칙을 만족하도록 정의해주세요. 
항상 변수가 왼쪽에 오거나, 오른쪽에 오도록 !